_sass/minimal-mistakes/_page.scss

```diff
@@ -594,3 +595,63 @@ body {
     }
   }
 }
+
+/* Media queries */
+
+@media screen and (max-width: 768px) {
+  .page__content, .archive__item {
+    font-size: 1em
+  }
+
+  .element-item {
+    width:calc(90% - 10px)!important;
+   }
+
+  .narrow p {
+    font-size: 0.9em!important;
+  }
+  .narrow {
+    font-size: 0.9em!important;
+  }
+
+  .page__meta.contributors {
+    font-size: .9em !important;
+  }
+
+  h2.archive__item-title, h3.archive__item-title {
+    font-size: 1.3em!important;
+  }
+
+
+  .feature__item--center .archive__item-body, .archive__item-body {
```

Comment:
Example of a element that should go in a different style sheet vs the page sheet.